skrypty/event_register.lua

```diff
@@ -0,0 +1,16 @@
+scripts.event_register = scripts.event_register or {
+    handlers = {}
+}
+
+function scripts.event_register:register_event_handler(id, event, callback, onetime)
+    local onetime = onetime or false
+    scripts.event_register:kill_event_handler(id)
+    scripts.event_register.handlers[id] = registerAnonymousEventHandler(event, callback, onetime)
```

Comment:
`id` eventow sa zapewnione unikatowo od mudleta, wiec moze ominac `id` tutaj i definiowac handler, ustawiac go w tablicy, i zwracac z funkcji?
W tym momencie w teorii jesli dwa moduly wysla taki sam ID to tylko jeden z nich bedzie aktywny.

Reply:
To jest to dokładnie o co mi chodzi, żeby rejestrować handler raz pod danym ID.